src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -60,6 +63,10 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
           height: 24px;
```

Comment:
`paper-icon-button` is no longer used

Reply:
Don't we use `paper-icon-button` inside a `paper-menu-button` ?
Or was it about the `paper-icon-buttons` I replaced in the list as you suggested in your other comment ?

Reply:
Yeah you use it inside it, but this style rules are not necessary anymore I think?

Reply:
Actually there is a 8px padding that shifts the text and it is not aligned with the rest of the table. The height reduces the size of the icon that is too big otherwise.